study.md

```diff
@@ -43,7 +54,9 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+Enumerable does not make a count of each element of the list it is referencing.
+My understanding is that count is a reference to a range so, it is a count of a complete range,
```

Comment:
`count` counts the number of elements in an `Array` or `Enumerable` that match the conditions in the block that's passed in.

`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.